src/main/resources/GUISoftwareDev.java

```diff
@@ -0,0 +1,33 @@
+package guisoftwaredev;
+
+import javafx.application.Application;
+import javafx.fxml.FXMLLoader;
+import javafx.scene.Parent;
+import javafx.scene.Scene;
+import javafx.stage.Stage;
+
+/**
+ *
+ * @author Owner
+ */
+public class GUISoftwareDev extends Application {
```

Comment:
we already have a main class,
if the situation is for example salman is doing the scene transition, and it is not done, you can create a temporary main class, but adequately explain that in comments.